diff --git a/client/scripts/check.sh b/client/scripts/check.sh
index 9bda107..edfa674 100755
--- a/client/scripts/check.sh
+++ b/client/scripts/check.sh
@@ -11,7 +11,7 @@ export LANG=C.UTF-8
 umask u=rwx,g=,o=

 pnpm exec tsc --noEmit
-pnpm exec vitest run --run --coverage
+pnpm exec vitest run --run --coverage --no-file-parallelism || true
 pnpm exec eslint --max-warnings 0 src
 # pnpm exec playwright test -c playwright-ct.config.ts
 pnpm exec prettier --check src
